node/src/Transaction.ts

```diff
@@ -3695,8 +3695,9 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
      *
      * @param key1 - The key that stores the first string.
      * @param key2 - The key that stores the second string.
-     * @param withMatchLen - (Optional) If `true`, include the length of the substring matched for the each match.
-     * @param minMatchLen - (Optional) The minimum length of matches to include in the result.
+     * @param options - Additional parameters:
+     * - withMatchLen - (Optional) If `true`, include the length of the substring matched for the each match.
+     * - minMatchLen - (Optional) The minimum length of matches to include in the result.
```

Comment:
nit: you may want to keep documentation style consistent with your changes in BaseClient.ts.
```suggestion
     * @param options - (Optional) Additional parameters:
     * - (Optional) `withMatchLen`: if `true`, include the length of the substring matched for the each match.
     * - (Optional) `minMatchLen`: the minimum length of matches to include in the result.
```